book/src/accounts/people.md

```diff
@@ -0,0 +1,118 @@
+# People Accounts
+
+A person represents a human's account in Kanidm. The majority of your users will be a person who
+will use this account in their daily activities. These entries may contain personaly identifying
+information that _is_ considered by kani to be sensitive. Because of this, there are default limits
```

Comment:
```suggestion
will use this account in their daily activities. These entries may contain personally identifying
information that _is_ considered by Kanidm to be sensitive. Because of this, there are default limits
```